django/test/testcases.py

```diff
@@ -825,6 +825,10 @@ class TransactionTestCase(SimpleTestCase):
     # are not available, we allow queries to be run.
     allow_database_queries = True
 
+    @classmethod
```

Comment:
Do you want to leave this method for now or revert to the old version to prevent further churn later?